rskj-core/src/main/java/co/rsk/rpc/modules/rsk/RskModuleImpl.java

```diff
@@ -143,7 +143,7 @@ private Block getByJsonBlockId(String id) {
         } else if ("latest".equalsIgnoreCase(id)) {
             return this.blockchain.getBestBlock();
         } else if ("pending".equalsIgnoreCase(id)) {
-            throw new JsonRpcUnimplementedMethodException("The method don't support 'pending' as a parameter yet");
+            throw RskJsonRpcRequestException.unimplemented("The method don't support 'pending' as a parameter yet");
```

Comment:
I'm not a big fan of this pattern, but I think we use to use _import static_ in this situations